docs/resources/cluster.md

```diff
@@ -152,7 +152,9 @@ Required:
 Optional:
 
 - `app_replication` (Boolean) Enables Argo CD state replication to the managed cluster that allows disconnecting the cluster from Akuity Platform without losing core Argocd features
+- `auto_agent_size_config` (Attributes) Autoscaler config for auto agent size (see [below for nested schema](#nestedatt--spec--data--auto_agent_size_config))
```

Comment:
I think we will need to add `auto` and `custom` to  https://github.com/akuity/terraform-provider-akp/blob/main/akp/data_source_akp_cluster_schema.go#L84 and
https://github.com/akuity/terraform-provider-akp/blob/main/akp/resource_akp_cluster_schema.go#L145